packages/ui-extensions/src/surfaces/checkout/api/standard/standard.ts

```diff
@@ -1790,6 +1812,12 @@ export interface CustomerPrivacy {
    * An object containing flags for each consent property denoting whether they can be processed based on visitor consent, merchant configuration, and user location.
    */
   allowedProcessing: AllowedProcessing;
+  /**
+   * Stored tracking consent metafield data.
+   *
+   * @example `[{key: 'analyticsType', value: 'granular'}, {key: 'marketingType', value: 'granular'}]`, `[]`, or `undefined`
+   */
+  metafields?: TrackingConsentMetafield[];
```

Comment:
@rcaplanshopify I think you mentioned there was some contention over metafields vs attributes. What was the outcome of that?

Reply:
The [current status](https://shopify.slack.com/archives/C05RV5EL5GE/p1714754248476499?thread_ts=1714750381.999309&cid=C05RV5EL5GE) is we are waiting to hear back from people who work on metafields.